node/third_party/openai-gpt-oss/metal/scripts/create-local-model.py

```
@@ -0,0 +1,358 @@
+    assert gain.dtype == torch.bfloat16
+    out_file.write(gain.view(torch.uint8).numpy().tobytes())
+
+
+def write_attn_sink(out_file, sink):
+    write_padding(out_file, alignment_multiple=16)
+
+    assert sink.dtype == torch.bfloat16
+    out_file.write(sink.view(torch.uint8).numpy().tobytes())
+
+
+def write_linear_weight(out_file, *args):
```

Ruff (0.14.10)

Missing type annotation for `*args`

(ANN002)